src/api/analytics.ts

```diff
@@ -49,9 +49,35 @@ export type AnalyticsOptions = {
     enrollmentDate?: string;
 };
 
+type KnownAscDescValues =
+    | "ouname"
+    | "programstatus"
+    | "createdbydisplayname"
+    | "lastupdatedbydisplayname"
+    | "enrollmentdate"
+    | "incidentdate"
+    | "lastupdated";
+
+type AscDescParameter = KnownAscDescValues | Id;
+
+type PageOptions = {
+    totalPages: boolean;
+    page: number;
+    pageSize: number;
+    paging: boolean;
+};
+
 export type GetEnrollmentsQueryOptions = {
     programId: Id;
-} & AnalyticsOptions;
+    programStatus?: "ACTIVE" | "COMPLETED" | "CANCELLED";
+    ouMode?: "DESCENDANTS" | "CHILDREN" | "SELECTED";
+    asc?: AscDescParameter;
+    desc?: AscDescParameter;
+    coordinatesOnly?: boolean;
+    headers?: string;
```

Comment:
`string` types are too unspecific. Docs "One or more headers name separated by comma". At the worst case, we can create an alias: "type HeadersSeparatedByCommas = string". Question: Are these headers known? (dx, dxname, pe, ...)